source/includes/_tipser-elements.md

```diff
@@ -132,16 +133,29 @@ To display `Product` in a compact view, add the `data-tipser-view="compact"` att
 
 ## `Collection` Element
 
-You can your store in collections. Each collection however can be rendered separately as a get the look widget or a simple product group, depending on your needs. It is all possible through `Collection` Element.
+You can render your collections as `Store`. Each collection, however, can be rendered separately as a simple product group, depending on your needs. It is all possible through the `Collection` Element.
 
 ```html
 <p name="My collection" data-tipser-cid="5b2788909d25801adcb23f4f" />
-<p name="My collection" data-tipser-cid="5b2788909d25801adcb23f4f" data-tipser-imgsize="1.2" />
+<p name="My collection" data-tipser-cid="5b2788909d25801adcb23f4f" data-tipser-imgsize="0.8" />
+<p name="My collection" data-tipser-cid="5b2788909d25801adcb23f4f" data-tipser-carousel />
+
 ```
 
-Elements with attribute `data-tipser-cid` will be replaced with `Collection` element of given id (value of `data-tipser-cid`). To make the collection items smaller / larger use the `data-tipser-imgsize` attribute with values `0.8` for smaller  and `1.2` for lager product tiles. The default value for imgSize parameter is `1`.
+Elements with attribute `data-tipser-cid` will be replaced with `Collection` element of given id (value of `data-tipser-cid`). 
+
+[![](collection.png)](/images/collcetion.png)
+
+To make the collection items smaller / larger use the `data-tipser-imgsize` attribute with values `0.8` for smaller  and `1.2` for lager product tiles. The default value for imgSize parameter is `1`. When changing the value to `0.8` you get slightly smaller product-cards:
+
+[![](collection-imgSize.png)](/images/collcetion-imgSize.png)
+
+If you'd like the collection of more than several products to take less space, you can display it as one-row only carousel element. To do that, please use `data-tipser-carousel` attribute.
 
-> [Open this snippet on Code Pen](https://codepen.io/tipser-tech/pen/YMMKMp)
+[![](collection-carousel.png)](/images/collcetion-carousel.png)
+
+
+> <a href="https://codepen.io/tipser-tech/pen/YMMKMp" target="_blank">Open this snippet on Code Pen</a>
```

Comment:
I guess you will need to update this codepen to really render the carousel component (currently it doesn't).